src/app/modules/authoring/learning-tasks/step-form/step-form.component.html

```diff
@@ -0,0 +1,107 @@
+<div fxLayout="column">
+
+    <mat-card fxFlex fxLayout="column" fxLayoutGap="20px" class="sticky">
```

Comment:
Same as above. We can create a smarter solution here.

Reply:
Same as above, to create new component for this part?